src/Components/JobsContainer/Internal/JobsDialog/JobsDialog.types.ts

```diff
@@ -1,14 +1,14 @@
 import { IStyle, IStyleFunctionOrObject } from '@fluentui/react';
 import { MockAdapter } from '../../../../Adapters';
 import JobsAdapter from '../../../../Adapters/JobsAdapter';
-import { IJob } from '../../../../Models/Constants/Interfaces';
+import { IAdtApiJob } from '../../../../Models/Constants/Interfaces';
 import { IExtendedTheme } from '../../../../Theming/Theme.types';
 
 export interface IJobsDialogProps {
     adapter: JobsAdapter | MockAdapter;
     isOpen: boolean;
     onClose: () => void;
-    onAddJob: (newJob: IJob, listOfJobs: IJob[]) => void;
+    onAddJob: (newJob: IAdtApiJob) => void;
```

Comment:
the components should not be passing around the API types. We should have a client type that we use internally. Today tehy will probably be very similar or identical, but it makes us resilient to them changing their structure.

Reply:
yeah good point. I just figured since this is the initial setup, i'll just use them interchangeably and fix it later.